logger/common.go

```diff
@@ -89,6 +87,12 @@ type Logger struct {
 	maxReadBytes int
 }
 
+// WindowsArgs struct for Windows configuration
+type WindowsArgs struct {
```

Comment:
should we move this to ```common_windows.go```?

Reply:
I have explored this option but I thought that including the struct here was the simplest solution since the struct is used only used if this condition `GOOS == "windows"` is true and multiple files with `_linux` with empty struct/ methods won't have to be created.